frontend/src/test/LoginPage.test.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import { create } from 'react-test-renderer';
+import { BrowserRouter } from 'react-router-dom';
+
+
+import LoginPage from '../Components/LoginPage';
+
+describe('Login Page Snapshot tests',()=>{
+    test(`Login Page matches snapshot`, () => {
+        const header = create(LoginPage);
```

Comment:
Just wanted to check shouldn't the header be assigned to create(<LoginPage/>)?

Reply:
`create(<LoginPage/>)`

Reply:
Yeah sorry, copy paste error